alternator/executor.cc

```diff
@@ -1486,7 +1486,7 @@ static future<executor::request_return_type> create_table_on_shard0(service::cli
             }
         }
```

Comment:
> Additionally, we need to consider the base schema version - when
we generate view updates the version of the base schema used for
reads should match the version of the base schema in view's base
info.
This is achieved by selecting the correct (old or new) schema in
`db::schema_tables::merge_tables_and_views` and using the stored
base schema in the schema_registry.

How does that work? Are we able to indirect through the renames when reading from an old schema?